src/composables/useActions.ts

```diff
@@ -33,6 +41,44 @@ export function useActions() {
     modalAccountOpen.value = true;
   }
 
+  async function createSpace(
+    networkId: NetworkID,
+    metadata: SpaceMetadata,
+    settings: SpaceSettings
+  ) {
+    if (!web3.value.account) {
+      forceLogin();
+      return false;
+    }
+
+    const network = getNetwork(networkId);
+    if (!network.managerConnectors.includes(web3.value.type as Connector)) {
+      throw new Error(`${web3.value.type} is not supported for this actions`);
+    }
+
+    const pinned = await network.helpers.pin(metadata);
+
+    const receipt = await network.actions.createSpace(auth.web3, {
+      controller: web3.value.account,
+      votingDelay: settings.votingDelay,
+      minVotingDuration: settings.minVotingDuration,
+      maxVotingDuration: settings.maxVotingDuration,
+      proposalThreshold: BigInt(settings.proposalThreshold),
+      qorum: BigInt(settings.quorum),
```

Comment:
This PR works well, there is just these `qorum` variables that need to be changed

Reply:
Those were renamed in sx.js, but it was not released yet: https://github.com/snapshot-labs/sx.js/pull/133

I was thinking there would be new deployment so I could release new version with deployment change (and this will be included in that release) so we avoid two small releases, but I can do special release for that fix too.